project2/styles.css

```
@@ -0,0 +1,178 @@
+/* FONTS
+	- default body text  sofia-pro, sans-serif;
+    - date & cite        orpheuspro, serif;
+*/
+
+/* DISPLAY SETTINGS & UTILITIES */
+
+* {
```

Since you're using pseudoelements, you'll want to update your box-sizing rule to make sure it applies to them, too:
```suggestion
*, *::before, *::after {
```